test/zeroD/test_zeroD.cpp

```diff
@@ -7,53 +7,57 @@
 
 using namespace Cantera;
 
-//This test ensures that prior reactor initialization of a reactor does not affect later integration within a network. This example was adapted from test_reactor.py::test_equilibrium_HP.
+// This test ensures that prior reactor initialization of a reactor does
+// not affect later integration within a network. This example was
+// adapted from test_reactor.py::test_equilibrium_HP.
 TEST(ZeroDim, test_individual_reactor_initialization)
 {
-    //Initial conditions
+    // Initial conditions
     double T0 = 1100.0;
     double P0 = 1013250;
     std::string X0 = "H2:1.0, O2:0.5, AR:8.0";
-    //Reactor solution, phase, and kinetics objects
+    // Reactor solution, phase, and kinetics objects
     std::shared_ptr<Solution> sol1 = newSolution("h2o2.yaml");
     std::shared_ptr<ThermoPhase> gas1 = sol1->thermo();
     std::shared_ptr<Kinetics> kin1 = sol1->kinetics();
     gas1->setState_TPX(T0, P0, X0);
-    //Set up reactor object
+    // Set up reactor object
     Reactor reactor1;
     reactor1.setKineticsMgr(*kin1);
     reactor1.setThermoMgr(*gas1);
-    //initialize reactor at arbitrary value
+    // Initialize reactor at arbitrary value prior to integration to ensure no impact
     reactor1.initialize(0.1);
-    //Setup reactor network and integrating
+    // Setup reactor network and integrate
     ReactorNet network;
     network.addReactor(reactor1);
     network.advance(1.0);
-    //Secondary gas for comparison
+    // Secondary gas for comparison
     std::shared_ptr<Solution> sol2 = newSolution("h2o2.yaml");
     std::shared_ptr<ThermoPhase> gas2 = sol2->thermo();
     std::shared_ptr<Kinetics> kin2 = sol2->kinetics();
     gas2->setState_TPX(T0, P0, X0);
     gas2->equilibrate("UV");
-    //Secondary reactor for comparison
+    // Secondary reactor for comparison
     Reactor reactor2;
     reactor2.setKineticsMgr(*kin2);
     reactor2.setThermoMgr(*gas2);
     reactor2.initialize(0);
-    //Get state of reactors
-    double state1 [reactor1.neq()];
-    double state2 [reactor2.neq()];
+    // Get state of reactors
+    double *state1 = new double[reactor1.neq()];
```

Comment:
Avoid raw pointers (use `std::uniqe_ptr` or `std::shared_ptr` instead)

Reply:
Why avoid raw pointers? The method takes a `double` pointer and fills that double with the current state of the reactor. So, I don't believe `vector` will work here.

Reply:
Good question … it follows coding guidelines (`CONTRIBUTING.md`):

> Avoid manual memory management (i.e. `new` and `delete`), preferring to use standard library containers, as well as `std::unique_ptr` and `std::shared_ptr` when dynamic allocation is required.

For `std::vector`, just pass the memory it owns (see [here](https://www.cplusplus.com/reference/vector/vector/data/)). That approach is used extensively elsewhere in Cantera.

Reply:
@anthony-walker … thanks for taking care of this comment. For my own feedback, that’s about it (although the `EXPECT_NEAR` loop could be simplified). But that’s really super minor.

Reply:
@ischoegl I appreciate the constructive feedback. It has been a learning experience that I can apply to my larger pull request.